File: CPP/Aksesoris.cpp
#include <iostream>
#include <string>
#include "PetShop.cpp"

using namespace std;

class Aksesoris : public PetShop{
    private:
        string jenis;
        string bahan;
        string warna;
    
    public:
        Aksesoris(){
            this->jenis = "";
            this->bahan = "";
            this->warna = "";
        }

        Aksesoris(int id, string nama_produk, int harga_produk, int stok_produk, string jenis, string bahan, string warna){
            this->setID(id);
            this->setNama_produk(nama_produk);
            this->setHarga_produk(harga_produk);
            this->setStok_produk(stok_produk);
            this->jenis = jenis;
            this->bahan = bahan;
            this->warna = warna;
        }

        void setJenis(string jenis){
            this->jenis = jenis;
        }
        void setBahan(string bahan){
            this->bahan = bahan;
        }
        void setWarna(string warna){
            this->warna = warna;
        }

        string getJenis(){
            return this->jenis;
        }
        string getBahan(){
            return this->bahan;
        }
        string getWarna(){
            return this->warna;
        }

        ~Aksesoris(){
        }



};

File: PHP/Aksesoris.php
<?php
require_once "PetShop.php";

class Aksesoris extends PetShop {
    protected $jenis;
    protected $bahan;
    protected $warna;

    public function __construct($id = 0, $nama_produk = "", $harga_produk = 0, $stok_produk = 0, $foto = "",  $jenis = "", $bahan = "", $warna = "") {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }
    // setter and getters
    public function setJenis($jenis) { $this->jenis = $jenis; }
    public function setBahan($bahan) { $this->bahan = $bahan; }
    public function setWarna($warna) { $this->warna = $warna; }

    public function getJenis() { return $this->jenis; }
    public function getBahan() { return $this->bahan; }
    public function getWarna() { return $this->warna; }

}


?>